components/shotAccordion.js

```diff
@@ -118,6 +124,10 @@ function ShotAccordion(props) {
                   target: props.shot["target"],
                   carryDiff: props.shot["carryDiff"],
                 });
+              case "strokesTaken":
```

Comment:
This is also handled in the default case

Reply:
I guess it's strokesTaken vs strokes. In this case, why not pick one?

Reply:
So do you want me to chnage it to strokes rather than strokesTaken?

Reply:
ye